app/assets/stylesheets/categories.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the categories controller here.
```

Comment:
remove this file if you are not adding any css
same for all the css and js files